src/fhir_mcp_server/server.py

```
@@ -283,42 +313,51 @@ async def search(
-                return await get_operation_outcome_required_error("type")
+                return format_response(
+                    await get_operation_outcome_required_error("type"), format
+                )
 
             client: AsyncFHIRClient = await get_async_fhir_client()
             async_resources: list[Any] = (
                 await client.resources(type).search(Raw(**searchParam)).fetch_raw()
             )
-            logger.debug("Async resources fetched:", async_resources) 
-            return async_resources
+            logger.debug("Async resources fetched:", async_resources)
```

Ruff (0.15.12)

[error] Too many arguments for `logging` format string

(PLE1205)